web/src/components/sections/GetNotified.tsx

```diff
@@ -64,6 +65,27 @@ export function GetNotified({
 }: {
   ctaRef: React.RefObject<HTMLDivElement>
 }) {
+  const [email, setEmail] = React.useState<string>('')
+
+  const signUpEmail = async (submittedEmail: string) => {
+    const data = {
+      email: submittedEmail,
+    }
+    // TODO(teddy): Add some email validation here
+    const response = await fetch(`${API_BASE}/notify`, {
```

Comment:
you no longer an axios guy? 🐑

Reply:
Eh no need to use axios here, just bloats the app a little it.